src/main/scala/org/globalforestwatch/summarystats/gladalerts/GladAlertsGridSources.scala

```diff
@@ -34,41 +35,41 @@ case class GladAlertsGridSources(gridTile: GridTile) extends GridSources {
   val intactForestLandscapes2016 = IntactForestLandscapes2016(gridTile)
   val braBiomes = BrazilBiomes(gridTile)
 
-  def readWindow(window: Extent): Either[Throwable, Raster[GladAlertsTile]] = {
+  def readWindow(windowKey: SpatialKey, windowLayout: LayoutDefinition): Either[Throwable, Raster[GladAlertsTile]] = {
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)